src/config.ts

```diff
@@ -39,11 +39,11 @@ export const config: Config = {
 // a new one will be generated every time the process starts.
 if (!config.did) {
   console.log('Creating an ephemeral DID.....')
-  const DidIon = await DidIonMethod.create({
+  const DidDht = await DidDhtMethod.create({ publish: true,
```

Comment:
it will publish by default, so no need for publish: true, unless you want to make it explicit